staging/vhost-device-gpu/src/device.rs

```diff
@@ -358,10 +358,7 @@ impl VhostUserGpuBackendInner {
                 let response_result = self.process_gpu_command(virtio_gpu, &mem, ctrl_hdr, gpu_cmd);
                 // Unwrap the response from inside Result and log information
                 response = match response_result {
-                    Ok(response) => {
-                        trace!("GpuCommand {cmd_name} success: {response:?}");
```

Comment:
I would personally keep this, but only do  ` trace!("GpuCommand {cmd_name} success");`,  but you can remove this if you disagree. I'm fine either way.

Reply:
i think this was just a duplication of https://github.com/dorindabassey/vhost-device/blob/8626bbd8a608af1a7b05c4cbaafaa27e0dc6940e/staging/vhost-device-gpu/src/device.rs#L153